cmd/xl-storage.go

```diff
@@ -387,6 +389,25 @@ func (s *xlStorage) CrawlAndGetDataUsage(ctx context.Context, cache dataUsageCac
 				oi:          oi,
 			})
 			if !version.Deleted {
+				// Bitrot check local data
+				if size > 0 && item.heal && opts.Bitrot {
+					s.waitForLowActiveIO()
+					err := s.VerifyFile(ctx, item.bucket, item.objectPath(), version)
```

Comment:
you don't need to do this VerifyFile automatically is done in HealObject() when ScanMode == DeepScan - the function disksWithAllParts() does this anyways. 

So why not just blindly call HealObject() with DeepScan options?

Reply:
@harshavardhana That is not the same. This will only check local disk data, which is intended. `HealObject` will check all disks. This way we keep track of the load we put on the system and will not 'disturb' others.

That is also why we keep the probability the same and we check on-disk data on every run and don't need a separate probability.